(DOCSP-33533) Follow-up small edits (#170)
mongodb/docs-meta: source/style-guide/style/links-and-cross-references.txt

Links and Cross-References
==========================

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2

>    :class: singlecol

Reviewer: Why did you delete this?

Author: Deleted by mistake, restored.

Avoid Inline References in Tutorials
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Avoid adding inline textual references in tutorials or stepped procedures. 

> If users follow inline references while in the middle of a procedure,
> they might have difficulty navigating back and reorienting themselves
> with their current step.

Reviewer 2: ```suggestion
If users follow inline references while in the middle of a procedure,
they might have difficulty navigating back to the procedure and reorienting themselves.
```
Suggestion for clarity

> Inline textual references may be appropriate for more reference-heavy pages.

Reviewer: I think a single paragraph is appropriate here.

Reviewer: Unresolved.

Minimize Repeated Links
~~~~~~~~~~~~~~~~~~~~~~~

- Don't link to the same target multiple times in the same section.

  You may link to the same target multiple times on the same page, but
  think carefully about whether it's absolutely necessary. For example,
  if a page is particularly long and contains many sections, then it might

>   be helpful to link to a target an additional time or two. Users may

Reviewer: ```suggestion
  help to link to a target an additional time or two. Users might
```

Reviewer: Unresolved.

Author: Addressed.

Avoid Inline References in Tutorials
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- Avoid adding inline textual references in tutorials or stepped procedures. 

  If users follow inline references while in the middle of a procedure,
  they might have difficulty navigating back and reorienting themselves
  with their current step.

>   Inline textual references may be appropriate for more reference-heavy pages.

Reviewer: Remove bullet, now that we have only one item.

Minimize Repeated Links
~~~~~~~~~~~~~~~~~~~~~~~

- Don't link to the same target multiple times in the same section.

  You may link to the same target more than once on the same page, but
  think carefully about whether it's absolutely necessary. For example,
  if a page is long and contains many sections, then it might be helpful
  to link to a target once or twice from this page to help users who
  navigate to a section far down on a page and may miss the initial

>   reference at the top of the page.

Reviewer: I think I still prefer the original wording for this bit.

Author: I reverted it. I think it is a bit too wordy, but I will go with your preference!

Avoid Inline References in Tutorials
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- Don't add inline textual references in tutorials or stepped procedures. 

  If users follow inline references while in the middle of a procedure, this complicates navigating back and reorienting themselves with
  their current step.

  Inline textual references may be appropriate for more reference-heavy pages,
  or tasks that aren't part of a streamlined tutorial for a particular action.

- Don't stack multiple admonitions directly after one another, as these
  can be distracting for users. To avoid stacking admonitions, group
  supplemental information from admonitions into a dedicated section.

- Don't use admonitions in tables and option lists, if possible, because
  admonitions reduce scannability of tables and options lists.

Minimize Repeated Links
~~~~~~~~~~~~~~~~~~~~~~~

- Don't link to the same target multiple times in the same section.

>   If a page is long and contains many sections, you may link to a target
>   once or twice from this page. This helps users who navigate to a section
>   far down on a page and may miss the initial reference at the top of the
>   page. However, don't link to a target more than twice on the same page.

Reviewer: I think this is one example scenario, but might not be the only scenario where this would be appropriate.

```suggestion
  You may link to the same target multiple times on the same page, but
  think carefully about whether it's absolutely necessary. For example, if
  a page is particularly long and contains many sections, then it might be
  helpful to link to a target an additional time or two. Users may
  immediately navigate to a section far down on a page and miss the
  initial reference.
```
